test/unit/polygon.js

```diff
@@ -106,7 +106,7 @@
     assert.deepEqual(polygon._getTransformedDimensions(), round, 'dims should match');
   });
 
-  QUnit.todo('polygon with exactBoundingBox true and skew', function (assert) {
+  QUnit.test('polygon with exactBoundingBox true and skew', function (assert) {
```

Comment:
therefore, removed todo, and it should be tested. Was todo because it was in between PRs